composer.json

```diff
@@ -62,7 +62,7 @@
         "amazonwebservices/aws-sdk-for-php": "required to use AWS version 1 cache resolver",
         "aws/aws-sdk-php": "required to use AWS version 2/3 cache resolver",
         "doctrine/mongodb-odm": "required to use mongodb-backed doctrine components",
-        "enqueue/enqueue-bundle": "add if you like to process images in background",
+        "enqueue/enqueue-bundle": "^0.6|^0.7 add if you like to process images in background",
```

Comment:
well you should then make the enque dependency `^0.6` or lift the suggestion to `^0.7`